src/Containers/CircleTable.tsx

```diff
@@ -19,135 +31,241 @@ export interface ICircleTable {
+                counter={index + 1}
+                key={generateKey(item.position + index)}
+                position={item.position}
+            >
+                <Chair
+                    relativeSize={relativeSize}
+                    position={item.position}
+                    occupiedBy={item.occupiedBy}
+                    isSeated={item.isSeated}
+                    isVisible={item.isVisible}
+                    isRound={item.isRound}
+                />
+            </ChairWrapper>
+        ));
+    };
+
+    /**
+     * Generates a unique key based on a string and a random number
+     * @param pre - a string to append to random number
+     * @returns {string} a unique key
+     */
+    const generateKey: generateKeyType = (pre) => {
+        return `${pre}_${Math.random()}`;
+    };
+
+    // Calculate the tangent based on the number of chairs in the array
+    const tan = Math.tan(Math.PI / chairs.length);
 
     return (
-        <>
-            <TableBody {...props}>
-                <RowMargin0>
-                    <Col6P0>
-                        <RowM0H25 />
-                        <RowM0H25 />
-                        <RowM0H25>
-                            <Col6MxAutoMt5>
-                                <TextWhiteDiv>{tableID}</TextWhiteDiv>
-                                <TextWhiteDiv>{partyName}</TextWhiteDiv>
-                                <TextOccupancyColor
-                                    occupancyColor={getOccupancyColor()}
-                                >
-                                    {occupancyStatus}
-                                </TextOccupancyColor>
-                            </Col6MxAutoMt5>
-                        </RowM0H25>
-                    </Col6P0>
-                    <Col4P0 />
-                </RowMargin0>
+        <div {...props}>
+            <TableBody
+                relativeSize={relativeSize}
+                numOfChairs={chairs.length}
+                chairTableBackground={colors.chairTableBackground}
+                tangentValue={tan}
+                occupancyColor={getOccupancyColor()}
+            >
+                {getChairs(chairs, relativeSize)}
+
+                <TableInfo>
+                    <div>
+                        {tableID}
+                        <br />
+                        {partyName}
+                        <br />
+                        <Status occupancyColor={getOccupancyColor()}>
+                            {occupancyStatus}
+                        </Status>
+                        <br />
+                    </div>
+                </TableInfo>
             </TableBody>
-        </>
+        </div>
     );
 };
 
+type Position = 'top' | 'bottom' | 'left' | 'right';
+
+type getPositionValueType = (position: Position) => number;
+
+type getTurnValueType = (
+    counter: number,
+    numOfChairs: number,
+    position: Position,
+) => string;
+
+const minChairsBeforeTableResize = 4;
```

Comment:
MIN_CHAIRS_BEFORE_TABLE_RESIZE
MIN_CHAIRS_TABLE_RESIZE_BREAKPOINT 
use SCREMAING_SNAKE_CASE for consts always when static const (Never Changing)